dietpi/dietpi-software

```diff
@@ -7079,6 +7085,104 @@ _EOF_
 			/boot/dietpi/func/create_mysql_db wordpress wordpress "$GLOBAL_PW"
 		fi
 
+		if To_Install 141 # Hugo
+		then
+			hdir=/mnt/dietpi_userdata/hugo
+			G_AGI hugo
+			G_EXEC mkdir -p "$hdir"
+			Create_User -d "$hdir" hugo
+
+			G_EXEC cd "$hdir"
+			if ! [[ -d "${hdir}/mysite" ]]; then
+				G_EXEC hugo new site mysite -f toml
+			fi
+			G_EXEC cd "${hdir}/mysite"
+			if ! [[ -d themes/PaperMod ]]; then
+				Download_Install "https://github.com/adityatelange/hugo-PaperMod/archive/master.tar.gz" "${hdir}/mysite"
+				G_EXEC mv "${hdir}/mysite/hugo-PaperMod-master" "${hdir}/mysite/themes/PaperMod"
+			fi
+
+			## Create a default config with a default theme
+			cat << _EOF_ > config.toml
+baseURL = 'http://$(G_GET_NET ip)'
+title = 'DietPi Test Blog'
+languageCode = 'en-us'
+theme = "PaperMod"
+
+[markup.goldmark.renderer]
+  unsafe = true
+
+_EOF_
+			## Create a systemd file
+			cat << '_EOF_' > /etc/systemd/system/hugo.service
+[Unit]
+Description=Hugo (DietPi)
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+SyslogIdentifier=Hugo
+User=hugo
+LogsDirectory=hugo
```

Comment:
Does it really log to `/var/log/hugo`, or can it be configured to log to STDOUT instead?